### PR TITLE
Initial look

### PR DESCRIPTION
**Pull request by jsmitthimedhin**

### CODE REVIEW

File: client/src/components/Book/DeleteButton.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ function DeleteButton(props) {
 
     try {
       //Calls Delete Endpoint
-      const response = await fetch("/api/delete", {
```
Comment (jsmitthimedhin) [inline client/src/components/Book/DeleteButton.jsx:25]: Delete functionality wasn't working. Seems to be the missing URL

File: client/src/components/BookCard.jsx [hunk 2]
```diff
@@ -7,20 +7,20 @@ function BookCard(props) {
   const { id, title, author_fname, author_lname, isbn, user_fname, onSelect } =
     props;
 
+  const formatString = (str) => {
```
Comment (jsmitthimedhin) [inline client/src/components/BookCard.jsx:10]: Cleared up some repeat code with a function.

File: client/src/components/BurgerMenu.jsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ Modal.setAppElement("#root");
 
 function Burger() {
   const [isOpen, setIsOpen] = useState(false);
-  const auth = useAuth();
+  const { token, logOut } = useAuth();
```
Comment (jsmitthimedhin) [inline client/src/components/BurgerMenu.jsx:14]: Pulled the logOut function and token here so you don't have to run `useAuth()` twice

File: client/src/components/Book/BookCover.jsx
Comment (zach-monroe) [inline client/src/components/Book/BookCover.jsx:1]: There is not, I was using axios at the beginning of this project and switched to fetch and forgot to change this over. Good catch

**Pull request merged**